core/node/eth_watch/src/event_processors/interop_root.rs

```diff
@@ -59,6 +62,16 @@ impl EventProcessor for InteropRootProcessor {
             .map_err(DalError::generalize)
             .map_err(EventProcessorError::internal)?;
 
+        // We stop processing interop roots after migration from Gateway to L1 is initiated, as these cannot be executed.
+        let gateway_migration_state = transaction
+            .server_notifications_dal()
+            .get_latest_gateway_migration_notification()
+            .await
+            .unwrap();
+        if gateway_migration_state == Some(GatewayMigrationNotification::FromGateway) {
+            return Ok(events_count);
+        }
```

Comment:
it doesn't work without knowing the current settlement layer

Reply:
Nevertheless, i'd rather not implement this. 
but wait for the new l1 batch without interop roots during gateway migration

Reply:
> without interop roots during gateway migration

@Deniallugo just to make sure: 
- we should keep adding interop roots into DB
- stop adding them to Bootloader memory
- wait for l1Batch with no interop root when migrating to L1 ?